Std/Tactic/TryThis.lean

```diff
@@ -56,14 +144,189 @@ apply the replacement.
     let stxRange := doc.meta.text.utf8RangeToLspRange stxRange
     unless stxRange.start.line ≤ params.range.end.line do return result
     unless params.range.start.line ≤ stxRange.end.line do return result
-    let .ok newText := props.getObjValAs? String "suggestion" | panic! "bad type"
     let .ok range := props.getObjValAs? Lsp.Range "range" | panic! "bad type"
-    result.push {
-      eager.title := "Apply 'Try this'"
-      eager.kind? := "quickfix"
-      eager.isPreferred? := true
-      eager.edit? := some <| .ofTextEdit params.textDocument.uri { range, newText }
-    }
+    let .ok suggestions := props.getObjVal? "suggestions" | panic! "bad type"
+    let .ok suggestions := suggestions.getArr? | panic! "bad type"
+    suggestions.foldlM (init := result) fun result s => do
+      let .ok newText := s.getObjValAs? String "suggestion" | panic! "bad type"
+      result.push {
+        eager.title := "Try this: " ++ newText
+        eager.kind? := "quickfix"
+        eager.isPreferred? := true
```

Comment:
I wanted to check that it makes sense to have `eager.isPreferred? := true` for each suggestion; I tried changing it, and couldn't see any effect. (`Cmd+.``return` still worked fine.)

Reply:
I believe if you set `isPreferred? := true`, the lightbulb turns big and blue, and you can trigger it with a single key command (`shift-alt-.` for me), which doesn't even pull up the menu and just applies it directly.

Reply:
Ah, got it—it's `Cmd+option+.` for me. Looks like if there are multiple preferred options, it pulls up the menu instead of inserting it immediately; I'd best change this.

Reply:
Ok, fixed in 4723c3b.